### PR TITLE
Small fix: correctly pass arguments

### PR DESCRIPTION
**Pull request by ariematsliah**
One of the calls to read_frames was assigning a 'video' to 'skeleton'.

Thoughts on adding mypy in CI?

### CODE REVIEW

File: sleap/io/format/deeplabcut.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def does_write(self) -> bool:
     def read(
         cls, file: FileHandle, full_video: Optional[Video] = None, *args, **kwargs,
     ) -> Labels:
-        return Labels(labeled_frames=cls.read_frames(file, full_video, *args, **kwargs))
```
Comment (talmo) [inline sleap/io/format/deeplabcut.py:72]: Are we sure that skeleton doesn't ever get passed in here through the `**kwargs`?

**Review by talmo — APPROVED** [review]
Nice catch! LGTM, but see comment below.

mypy: Yes, please feel free to set this up in the CI workflow (`.github/workflows/ci.yml`) in another PR :). I imagine we'll have to do a lot of fixes to get that to pass.

File: sleap/io/format/deeplabcut.py (hunk 1 above)
Comment (ariematsliah) [reply sleap/io/format/deeplabcut.py:72]: not sure if I understand the question - the 'read_frames' method does not access args / kwargs

**Comment by ariematsliah** [comment]
re mypy sounds good - for now dropped a 'todo' under our project

File: sleap/io/format/deeplabcut.py (hunk 1 above)
Comment (ariematsliah) [reply sleap/io/format/deeplabcut.py:72]: but the method read_frames is not reading anything from kwargs, so this is a no-op right?

Comment (talmo) [reply sleap/io/format/deeplabcut.py:72]: It's not that `read_frames` is using the `**kwargs`, it's that it's using `skeleton`. Previously, if `kwargs = {"skeleton": [...]}` in `read()`, that would pass through to the named keyword arg in `read_frames()`. With this change, you cannot pass a skeleton to `read_frames()` through `read()`.

I don't think this is a serious use case, but to ensure it is equivalent to the previous behavior, `read()` should be:

```py
@classmethod
def read(cls, file: FileHandle, full_video: Optional[Video] = None, skeleton: Optional[Skeleton] = None, *args, **kwargs) -> Labels:
    return Labels(labeled_frames=cls.read_frames(file=file, full_video=full_video, skeleton=skeleton))
```

See what I'm saying?

Comment (ariematsliah) [reply sleap/io/format/deeplabcut.py:72]: Thanks for the explanation - adding them back

**Review by talmo — APPROVED** [review]
LGTM

**Pull request merged**